rpc/eth_api.go

```diff
@@ -264,6 +264,18 @@ func (e *PublicEthAPI) GetTransactionLogs(txHash common.Hash) ([]*ethtypes.Log,
 	return e.backend.GetTransactionLogs(txHash)
 }
 
+// ExportAccount exports an account's balance, code, and storage at the given block number
+func (e *PublicEthAPI) ExportAccount(address common.Address, blockNumber BlockNumber) (string, error) {
```

Comment:
can you add a note to deprecate this once we get the export genesis logic to work?